symbols/overlay29.yml

```diff
@@ -248,7 +248,7 @@ overlay29:
         
         Floor types:
           0 appears to mean the current floor is "normal"
-          1 appears to mean the current floor is a fixed floor
+          1 means the current floor is a fixed floor with Is Free Layout enabled
```

Comment:
"Is free layout" is not a vanilla setting, it's added by a patch. So the actual meaning has to be something else.

Reply:
In my testing, it returned 1 when it was a fixed floor with a Warp Zone and 0 otherwise; I assumed they were one and the same.